src/frontend/Navigation/Drawer.tsx

```diff
@@ -166,6 +170,13 @@ const DrawerContent = ({navigation}: DrawerContentComponentProps) => {
           <DrawerListItemIcon iconName="settings-suggest" />
           <ListItemText primary={<FormattedMessage {...m.appSettings} />} />
         </ListItem>
+        <ListItem
+          onPress={() => {
+            // TODO
+          }}>
+          <DrawerListItemIcon iconName="privacy-tip" />
```

Comment:
The designs show a slightly different icon, but that icon isn't present in the Material icons we use. I used the closest one I could find.